Dockerfile

```diff
@@ -1,9 +1,25 @@
-FROM scratch
+FROM golang:1.14-buster AS build
```

Comment:
If we do choose to build the application inside the multi-stage dockerfile you should use newer and smaller image `golang:1.15.2-alpine3.12` and make sure to build with the following envvars and commands:

```
ENV CGO_ENABLED="0" \
    GOOS="linux"

RUN go test ./... \
    && go build -a -installsuffix cgo -ldflags "-X main.version=${ESTAFETTE_BUILD_VERSION} -X main.revision=${ESTAFETTE_GIT_REVISION} -X main.branch=${ESTAFETTE_GIT_BRANCH} -X main.buildDate=${ESTAFETTE_BUILD_DATETIME}" -o ./publish/${ESTAFETTE_GIT_NAME} .
```